.github/workflows/full-ci.yml

```diff
@@ -51,7 +51,11 @@ jobs:
         run: sudo systemctl kill mono-xsp4 || true
 
       - name: Integration tests
-        run: mage integrationfull
+        env:
+          TT_ENABLE_COREDUMP_TESTS: '1'
+        run: |
+          sudo apt install -y gdb
```

Comment:
We have two actions:

https://github.com/tarantool/tt/tree/master/.github/actions/prepare-ce-test-env
https://github.com/tarantool/tt/tree/master/.github/actions/prepare-ee-test-env

For installing some stuff.

Reply:
Done.